cmd/bootstrap/run/result.go

```diff
@@ -7,12 +7,9 @@ import (
 
 func GenerateRootResult(block *flow.Block, commit flow.StateCommitment) *flow.ExecutionResult {
 	result := &flow.ExecutionResult{
-		ExecutionResultBody: flow.ExecutionResultBody{
-			PreviousResultID: flow.ZeroID,
-			BlockID:          block.ID(),
-			Chunks:           chunks.ChunkListFromCommit(commit),
-		},
-		Signatures: nil,
```

Comment:
Totally agree. I actually contemplated removing the  `Signatures` field entirely. But I wasn't sure either. So I just moved it up to the ExecutionReceipt. At least then we potentially have an unused field but it can't result in storage write-read inconsistencies anymore.

Reply:
removed `ExecutionReceipt.ResultSignature` entirely